src/d3a/models/market/__init__.py

```diff
@@ -135,6 +135,12 @@ def _is_constant_fees(self):
     def open_bids_and_offers(self, **kwargs) -> Tuple[List, List]:
         return list(self.bids.values()), list(self.offers.values())
 
+    def orders_per_slot(self) -> Dict[str, Dict]:
+        """Return all orders in the market per time slot."""
+        bids = [bid.serializable_dict() for bid in self.bids.values()]
+        offers = [offer.serializable_dict() for offer in self.offers.values()]
```

Comment:
Why are we converting with `serializable_dict()` here? Given the name of the method, I would expect to receive `Bid` and `Offer` instances.

Reply:
The actual requirement is a dictionary of time_slot: serialized orders
I have already added the return type hint, however if you think that we still have to change the name of the method to reflect that, let me know